customers.sql

```diff
@@ -0,0 +1,11 @@
+select a.id as customer_id,
+    a.name,
```

Comment:
the style guide linked in our homework suggests you use descriptive aliases for your tables